microsetta_interface/model_i18n.py

```diff
@@ -0,0 +1,70 @@
+from flask_babel import gettext
+import copy
+
+
+def translate_sample(sample):
+    i18n_sample = copy.deepcopy(sample)
+    i18n_sample["sample_site"] = gettext(sample["sample_site"])
+    return i18n_sample
+
+
+def translate_source(source):
+    i18n_source = copy.deepcopy(source)
+    i18n_source["source_type"] = gettext(source["source_type"])
+    return i18n_source
+
+
+def translate_survey_template(survey_template):
+    i18n_survey_template = copy.deepcopy(survey_template)
+    i18n_survey_template["survey_template_title"] = \
+        gettext(survey_template["survey_template_title"])
+    return i18n_survey_template
+
+
+def declare_enum_values():
+    # There are a few enum fields in the private api
+    # model object types that need to be translated
+    # We define those enum values here so that babel
+    # picks them up for translation.
+
+    # Sample.sample_site
+    #     sample_site:
+    #       enum: ["Blood (skin prick)", "Saliva", "Ear wax", "Forehead",
+    #       "Fur", "Hair", "Left hand", "Left leg", "Mouth", "Nares",
+    #       "Nasal mucus", "Right hand", "Right leg", "Stool", "Tears",
+    #       "Torso", "Vaginal mucus", null]
```

Comment:
would it be possible to do a `gettext` on `str(None)`, or something similar so that the null is reasonable for the user?

Reply:
Replaced None with "" in the two sample fields where they show up.